.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,15 @@
 <!--
+    Thank you for contributing!
+
     ESLint adheres to the [JS Foundation Code of Conduct](https://js.foundation/community/code-of-conduct).
 -->
 
-**What is the purpose of this pull request? (put an "X" next to item)**
+#### Prerequisites checklist
+
+- [ ] I have read the [contributing guidelines](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md).
+- [ ] The team has reached consensus on the changes proposed in this pull request. If not, I understand that the process will begin with this pull request and won't be merged until it has been accepted.
```

Comment:
Not critical, but could this say "I understand that the [evaluation/consensus] process will begin with this pull request and won't be merged until it has been accepted"?

Also, I think we're conflating "team has reached consensus" with "issue is accepted". This is correct from the perspective of our process, but a newcomer may not understand that intuitively. It would be great if we could use the same verb phrase throughout the sentence, to the extent it is feasible.

Reply:
Good idea! Updated - please let me know what you think.